matching-notification-resources/src/main/resources/resources/uicomponents/matchingNotification/notifiedMatchesTable.js

```diff
@@ -79,7 +79,7 @@ var PhenoTips = (function (PhenoTips) {
             var [successfulIds, failedIds] = this._utils.getResults(ajaxResponse.responseJSON.results);
 
             if (failedIds.length > 0) {
-                alert("Sending notification failed for the matches with the following ids: " + failedIds.join());
+                alert("$escapetool.javascript($services.localization.render('phenotips.matchingNotifications.matchesTable.onFailureAlert')) " + failedIds.join());
```

Comment:
+1, phenotips.matchingNotifications.matchesTable.onFailureAlert is present in ApplicationResources.properties